packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -36,12 +39,26 @@ import { ImageActor } from '../types/IActor';
  * which will add volumes to the specified viewports.
  */
 class VolumeViewport extends BaseVolumeViewport {
+  // Camera properties
+  private initialViewUp: Point3;
+
   private _useAcquisitionPlaneForViewPlane = false;
   constructor(props: ViewportInput) {
     super(props);
 
-    const { orientation } = this.options;
+    const camera = vtkCamera.newInstance();
 
+    this.initialViewUp = <Point3>[0, -1, 0];
+    const viewPlaneNormal = <Point3>[0, 0, -1];
+
+    camera.setDirectionOfProjection(
+      -viewPlaneNormal[0],
+      -viewPlaneNormal[1],
+      -viewPlaneNormal[2]
+    );
+    camera.setViewUp(...this.initialViewUp);
+
```

Comment:
why do you need this?

Reply:
From what I understand, this initialViewUp is needed to get the rotation of the of the camera at a given point in time. See the StackViewport, that's where I got this from.

Reply:
https://github.com/cornerstonejs/cornerstone3D/blob/19d7d73eca5190a6bd81429a5a1858c500946d61/packages/core/src/RenderingEngine/StackViewport.ts#L1076

https://github.com/cornerstonejs/cornerstone3D/blob/19d7d73eca5190a6bd81429a5a1858c500946d61/packages/core/src/RenderingEngine/StackViewport.ts#L243